Userland/Utilities/crypto.cpp

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2024, Ali Mohammad Pur <mpfard@serenityos.org>
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/HashMap.h>
+#include <AK/NumberFormat.h>
+#include <AK/Random.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/ElapsedTimer.h>
+#include <LibCrypto/AEAD/ChaCha20Poly1305.h>
+#include <LibCrypto/Authentication/GHash.h>
+#include <LibCrypto/Authentication/HMAC.h>
+#include <LibCrypto/Authentication/Poly1305.h>
+#include <LibCrypto/Checksum/Adler32.h>
+#include <LibCrypto/Checksum/CRC32.h>
+#include <LibCrypto/Cipher/AES.h>
+#include <LibCrypto/Cipher/ChaCha20.h>
+#include <LibCrypto/Forward.h>
+#include <LibCrypto/Hash/BLAKE2b.h>
+#include <LibCrypto/Hash/MD5.h>
+#include <LibCrypto/Hash/MGF.h>
+#include <LibCrypto/Hash/PBKDF2.h>
+#include <LibCrypto/Hash/SHA1.h>
+#include <LibCrypto/Hash/SHA2.h>
+#include <LibMain/Main.h>
+
+[[gnu::noinline]] void use(auto& value)
```

Comment:
`static`, and probably `NEVER_INLINE` instead of `[[gnu::noinline]]`

Reply:
There's `AK::taint_for_optimizer`

Reply:
Ah right, that was a thing.